app/assets/stylesheets/_variables.scss

```diff
@@ -76,3 +76,8 @@ $sidebar-fg-selected:           #bbb;
 $sidebar-fg-hover:              #fff;
 $sidebar-settings-fg:           $sidebar-fg-selected;
 $sidebar-settings-fg-selected:  #fff;
+
+/* Column */
+
+$btn-close-hover:               rgba(0, 0, 0, 0.3);
```

Comment:
It would be better to rename these variables to be more specific. Something like `$column-btn-close-hover`

Reply:
Ok!